input/core/src/commonMain/kotlin/majestic/editor/body/chunksUI/ImageChunk.kt

```diff
@@ -0,0 +1,22 @@
+package majestic.editor.body.chunksUI
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.dp
+import majestic.dragdrop.DragAndDropBox
+
+
+@Composable
+internal fun ImageChunk(
+    resource: Painter, labels: Labels
+) = DragAndDropBox(
+    modifier = Modifier
+        .fillMaxSize()
+        .height(114.dp),
```

Comment:
While this may still work, the modifier is semantically incorrect. instead of

```kotlin
Modifier.fillMaxSize().heigh(114.dp)
```

go for

```kotlin
Modifier.fillMaxWidth().height(114.dp)
```